### DISCUSSION TITLE
user-guide/faqs/monochrome-emoji

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#78] [文档评论区]
# user-guide/faqs/monochrome-emoji

近期许多同学在系统更新后发现 Zotero 中 emoji 变成黑白了，初步判断这是由于系统更新出现的问题。 emoji变成黑白 该问题可以通过修改 Zotero 设置 来解决 具体操作步骤如下： 1. 在菜单编辑选择“首选项”： 编辑器 2. 弹出如下图所示的对话框。 点击“I accept the risk” 同意承担风险 3. 在搜索框中输入 e...

https://zotero-chinese.com/user-guide/faqs/monochrome-emoji.html

**Comment by Ying-Zi66** [comment]
厉害问题解决了，但是标签的样式变了，有什么好的emoji库吗

**Reply by Oujinhai** [reply]
一样，蹲一个

**Reply by northword** [reply]
可以尝试一下 chikit 发现的另一种方法：https://zotero-chinese.com/user-guide/faqs/Emoji%E5%8F%98%E6%88%90%E4%BA%86%E9%BB%91%E7%99%BD%E8%AF%A5%E5%A6%82%E4%BD%95%E8%A7%A3%E5%86%B3.html#%E4%B8%80%E3%80%81%E5%AE%89%E8%A3%85-seguiemj-%E5%AD%97%E4%BD%93

**Comment by Lindsey-D** [comment]
我这样操作之后，只有部分标签恢复了颜色

**Comment by pp513** [comment]
厉害了！！noice！！

**Comment by DocBles** [comment]
太强了，感谢大佬！

**Comment by AlexChy829** [comment]
最近又有新问题了好像，颜色标签在左下角标签面板里全部显示的黑色不知道咋回事
